npm/ng-packs/packages/theme-shared/src/lib/services/status-code-error-handler.service.ts

```diff
@@ -90,8 +90,9 @@ export class StatusCodeErrorHandlerService implements CustomHttpErrorHandlerServ
 
         this.showConfirmation(title, message).subscribe();
         break;
-
       case 403:
+        this.authService.logout();
```

Comment:
But it might cause logout user, even unpermitted specific path e.g:

### Having permissions
- Role

### Scenario
Try to go `identity/users`

### Result
It'll logout just because of you haven't users permissions.